### PR TITLE
Add `keystone_branch` var to the zuul job

### LINKED ISSUE #18: Support defining keystone version
`keystone_git_install_branch` can be used to define keystone version in `openstack-ansible-os_keystone`

### PR DESCRIPTION
**Pull request by outcatcher**
Allow selecting Keystone version to be deployed.

Add zuul jobs for the supported OpenStack releases.

Resolve  #18

### CODE REVIEW

**Review by lego963 — DISMISSED** [review]

**Review by lego963 — DISMISSED** [review]

File: ci/playbooks/pre.yaml [hunk 1]
```diff
@@ -30,7 +30,11 @@
         chdir: "{{ repo_tmp }}"
       shell: >
         ansible-playbook
-        -e "provider='{{ provider }}' vm_name='{{ vm_name }}' seed='{{seed}}' result_clouds='{{ result_clouds }}'"
+        -e "provider='{{ provider }}'
+        vm_name='{{ vm_name }}'
+        seed='{{seed}}'
+        keystone_git_install_branch='{{ keystone_branch | default('master') }}'
```
Comment (lego963) [inline ci/playbooks/pre.yaml:36]: master?

File: zuul.d/01_jobs_tox.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+
```
Comment (lego963) [inline zuul.d/01_jobs_tox.yaml]: blank line

**Review by lego963 — CHANGES_REQUESTED** [review]

File: ci/playbooks/pre.yaml (hunk 1 above)
Comment (outcatcher) [reply ci/playbooks/pre.yaml:36]: It's a default keystone branch: https://opendev.org/openstack/keystone/branches

**Review by lego963 — APPROVED** [review]

**Comment by outcatcher** [comment]
regate

**Pull request merged**